chore: update bootstrap to ^4.0.0-alpha.5

jnizet opened the pull request:
Closes #915

BREAKING CHANGE: ng-boostrap now uses bootstrap 4.0.0-alpha.5, which modified the markup used to generate tooltip and popover arrows (see https://github.com/twbs/bootstrap/pull/17238). Using ng-bootstrap with an older version of bootstrap will thus not display tooltip and popover arrows anymore.

pkozlowski-opensource commented:
From my quick testing datepicker was a bit broken as well. Or was it only my env?

jnizet commented:
I'll check that.

jnizet commented:
There is indeed a small visual glitch in the datepicker navigation. I'll investigate. If anyone has an idea...
<img width="217" alt="capture d ecran 2016-10-19 a 19 16 58" src="https://cloud.githubusercontent.com/assets/1608223/19529573/c51bc906-9630-11e6-867a-ad419b6329bc.png">

pkozlowski-opensource commented:
I was just wondering if we can keep the `<div class="tooltip-arrow"></div>` element around for a release or 2 to avoid breaking changes. If I understand the change correctly it wouldn't hurt (I mean - there should be no visual effect, right?).

jnizet commented:
I've added a css rule to the navigation select to keep its original height, and reverted the removal of the .tooltip-arrow and .popover-arrow divs, but added a TODO to remember removing them later.

pkozlowski-opensource commented:
@jnizet thnx, this looks good. I'm planning to cut another release tomorrow and after I will re-test entire site again with your fix.

pkozlowski-opensource commented:
Just noticed one more thing - there is sth wrong with the header: gh and twitter buttons got pulled to the left instead staying on the right. There were many changes to the navbar so we are probably hitting one of them.

jnizet commented:
I'll try looking at this tomorrow. Sorry I completely missed that.

jnizet commented:
@pkozlowski-opensource I couldn't reproduce this problem. It does look a little bit different in small screen sizes (where it doesn't look nice in both old and new versions), but I can't see the buttons going to the left as you described.

pkozlowski-opensource commented:
@jnizet sorry for the noise, I had a messed-up env when I saw "the bug". It is all good, merged. Thnx!
